huaweicloud/services/css/resource_huaweicloud_css_thesaurus.go

```diff
@@ -0,0 +1,225 @@
+package css
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/css/v1/thesaurus"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func ResourceCssthesaurus() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: ResourceCssthesaurusCreate,
+		ReadContext:   ResourceCssthesaurusRead,
+		UpdateContext: ResourceCssthesaurusCreate,
```

Comment:
Why use `ResourceCssthesaurusCreate`? If update is the same as create,  suggest to also write a updating mtheod which call the creating method internally.